src/main/resources/messages/messages.properties

```diff
@@ -11,6 +11,7 @@ help=Show this help message and exit
 no-progress=Disable progress on executed command
 no-colors=Disable colors and styles
 verbose=Provide more information on the command execution
+fail-if-incomplete=Return error code if not fully translated or approved
```

Comment:
We should create separate messages for the status subcommands. 

For `crowdin status translation`: `Fail execution if the current project is not fully translated`

For `crowdin status` and `crowdin status proofreading` - `Fail execution if the current project is not fully approved`